tests/state/test_get_list_state.py

```diff
@@ -0,0 +1,389 @@
+    ])
+
+    result_data = json.loads(result.output).get('result')[0]
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_data
+
+
+def test_get_list_state_with_invalid_limit():
+    """
+    Case: get a state by its invalid limit.
+    Expect: the following limit is invalid error message.
+    """
+    invalid_limit = '-3'
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--limit',
+        invalid_limit,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': {
+            'limit': [
+                f'Limit must be greater than 0.',
+            ],
+        },
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_state_with_non_existing_limit():
+    """
+    Case: get a state by its non-existing account address starting from.
+    Expect: block not found error message.
+    """
+    non_existing_limit = 1e+10**2
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--limit',
+        non_existing_limit,
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    expected_error_message = {
+        'errors': f'Invalid limit count.',
+    }
+
+    assert FAILED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert dict_to_pretty_json(expected_error_message) in result.output
+
+
+def test_get_list_state_with_reverse():
+    """
+    Case: get a list of states by reverse.
+    Expect: reverse list of a states are returned.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+        '--reverse',
+        '--node-url',
+        NODE_IP_ADDRESS_FOR_TESTING,
+    ])
+
+    result_data = json.loads(result.output).get('result')[0]
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert result_data
+
+
+def test_get_list_state_without_node_url(mocker):
+    """
+    Case: get a list of states by its address without passing node URL.
+    Expect: states are returned from node on localhost.
+    """
+    expected_result = {
+        "data": [
+            {
+                "address": "0000000000000000000000000000000000000000000000000000000000000000000001",
+                "data": "CAE=",
+            },
+        ],
+    }
+
+    mock_get_state_by_address = mocker.patch('cli.state.service.loop.run_until_complete')
+    mock_get_state_by_address.return_value = expected_result
+
+    runner = CliRunner()
+    result = runner.invoke(cli, [
+        'state',
+        'get-list',
+    ])
+
+    result_data = json.loads(result.output).get('result')
+
+    assert PASSED_EXIT_FROM_COMMAND_CODE == result.exit_code
+    assert expected_result.get('data') == result_data
+
+
+def test_get_list_state_with_invalid_node_url():
```

Comment:
This test is for a non-existing node URL, but not an invalid node URL. Update documentation.
```suggestion
def test_get_batch_status_with_non_existing_node_url():
```
Create test for invalid node URL.

Reply:
@anastasiia-bilova, done.